irohad/ametsuchi/impl/rocksdb_specific_query_executor.cpp

```diff
@@ -345,18 +369,47 @@ RocksDbSpecificQueryExecutor::readTxs(
   uint64_t remains = query.paginationMeta().pageSize() + 1ull;
```

Comment:
Why Qry is not a strict type but template?

Reply:
See ```readTxs``` call places plz.